### ISSUE TITLE
Mutators are called multiple times with the outcome of the previous call as its next value

### ISSUE THREAD

**Issue by petervmeijgaard** [vuex-orm/vuex-orm#744]
#### Describe the bug
Mutators are called multiple times with the outcome of the previous call as its next value.

I want to create a mutator that transforms the timestamps returned from the API I'm consuming into a Moment variable. This doesn't seem to work with the normal way how mutators are defined. The first time when it mutates the value, I get a valid Moment variable, but it's mutating a second time and now uses the newly created Moment variable as its property. This shouldn't be the case.

#### Steps to reproduce the bug

1. Define a model with a mutator that returns an instance of Moment
2. Create one or multiple instances of that model
3. Retrieve the created model(s)
4. See an error, saying: `config._d.getTime is not a function`

It seems that it's calling the mutator multiple times instead of just once. The first time it's being called, the correct moment instance is returned, but then it's called again; now with the newly created moment instance.

#### Expected behavior

I would expect that the result is a valid Moment variable that I can use and that the mutator is being called once.

#### Versions

- Vuex ORM: 0.36.4
- Vue: 2.6.11

#### Link to minimal reproduction

- Link to [CodeSandbox](https://codesandbox.io/s/vuex-orm-mutators-problem-l5yk4)

As you can see in the console, it calls the mutator twice. The first time with the correct value, but the second time it's using the moment variable instead of the previous value.

**Comment by psarin** [comment]
Same problem and I'm seeing the exact same behavior as noted above. The last working version appears to be 0.33.0 .

**Comment by SimonDemeyereJobtoolz** [comment]
@petervmeijgaard did you every fix this? I am struggling with the same issue.

**Comment by petervmeijgaard** [comment]
@SimonDemeyereJobtoolz No, I gave up. This project is abandoned as the last commit to master was three years ago. I would suggest looking for an alternative library than `vuex-orm`
